src/components/NFTBook/ItemCard.vue

```diff
@@ -152,10 +152,63 @@
             :class="['text-[24px]', 'font-600', titleStyle]"
             v-text="NFTName"
           />
-          <p :class="['text-14', 'whitespace-pre-line', descriptionStyle]">
+          <p
+            v-if="!isDetailsPreset"
+            :class="['text-14', 'whitespace-pre-line', descriptionStyle]"
+          >
             {{ bookDescriptionTrimmed }}
           </p>
-          <NFTBookSpecTable class="mt-[12px]">
+          <template v-else>
+            <div
+              v-if="isDetailsPreset"
+              :class="[
+                'relative',
+                'h-full',
+                { 'max-h-[240px]': !isExpanded },
+                'overflow-y-hidden',
+                'flex flex-col',
+                'gap-[40px]',
+              ]"
+            >
+              <p
+                :class="[
+                  'blick',
```

Comment:
what is `blick` class?

Reply:
block la~
[💄 Update styles](https://github.com/likecoin/liker-land/pull/2015/commits/e93bde4bbd1b94b17086939d31519eb6dc70601c)